SU-HSDO/suhumsci
# SHS-5061: Vertical link card link styles

# READY FOR REVIEW

## Summary
- Update styles of description links on vertical link cards to match [requirement](https://app.clickup.com/t/36718269/SHS-5063).

## Need Review By (Date)
10/26

## Urgency
Medium

## Steps to Test
- Visit the site in Colorful theme.
- Visit a page with Vertical Links Cards, like here: http://hs-colorful.suhumsci.loc/components/collections-cards
- Add a regular link, a private link, a file link and a PDF link
- Ensure that links are white with gray hover/focus. Ensure that the icons match the link text color
- Switch to view a [site in Traditional theme](http://hs-traditional.suhumsci.loc/components/collections-cards) and repeat the steps above. Make sure that the links are black, wit a dark gray hover/focus.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
